custom/layouts/partials/nav.ejs

```diff
@@ -0,0 +1,38 @@
+<div class="g-upper-body">
```

Comment:
do you need this custom file? it doesn't look like you ended up changing anything in it

Reply:
Actually, you do.  What I have found is that if you change a child ejs file, you have to include the parent ejs, otherwise it doesn't include the custom code.  It has to do with some pathing issue.  In this instance, if I don't include the nav.ejs, the search filter doesn't get included in the search results page.  I can create a ticket for this.